tests/integration/model_test.py

```diff
@@ -142,12 +138,16 @@ def test_find_activate():
                                             |                               |                       |
         request date            20150906    |                               |                       |
         """
-
         rtu = TripUpdate.find_by_contributor_period([COTS_CONTRIBUTOR_ID], datetime.date(2015, 9, 6))
+        expected_result = [
+            "70866ce8-0638-4fa1-8556-1ddfa22d09d3",
+            "70866ce8-0638-4fa1-8556-1ddfa22d09d4",
+            "70866ce8-0638-4fa1-8556-1ddfa22d09d5",
+        ]
         assert len(rtu) == 3
-        assert rtu[0].vj_id == "70866ce8-0638-4fa1-8556-1ddfa22d09d3"
-        assert rtu[1].vj_id == "70866ce8-0638-4fa1-8556-1ddfa22d09d4"
-        assert rtu[2].vj_id == "70866ce8-0638-4fa1-8556-1ddfa22d09d5"
+        result = [rtu[0].vj_id, rtu[1].vj_id, rtu[2].vj_id]
+        result.sort()
+        assert bool(result == expected_result) == True
```

Comment:
Wouldn't this work? (same below)
```suggestion
        assert rtu.sort() == expected_result
```
The bool cast and the intermediate result look useless.